<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormConditionStd</class>
 <widget class="QWidget" name="FormConditionStd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frameHeader
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #a6a6a6, stop: 0.08 #7f7f7f, stop: 0.39999 #717171, stop: 0.4 #626262, stop: 0.9 #4c4c4c, stop: 1 #333333);
	padding: 5px;
	border: 1px solid black;
	border-top-left-radius: 8px;
	border-top-right-radius: 8px;
	border-bottom: none;
}
#FormConditionStd
{
	border: 1px solid black;
	border-radius: 8px;
}
#FormConditionStd QLabel
{
	color: white;
}
#frameHeader #labelTitle
{
	font-weight: bold;
	padding-right: 15px;
	border: 1px solid rgb(75, 75, 75);
	border-top: none;
	border-bottom: none;
	border-left: none;
}
#frameContent
{
	background-color: rgb(78, 78, 78);
	padding: 5px;
	border: 1px solid black;
	border-top: none;
	border-bottom-right-radius: 8px;
	border-bottom-left-radius: 8px;
}
#frameContent .buttonStyled
{
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 20px;
    min-height: 16px;
}
#frameContent .buttonStyled:hover
{
	background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4, radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#frameContent .buttonStyled:pressed
{
	background: qradialgradient(cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1, radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}
#frameContent QComboBox, #frameContent QLineEdit
{
	border: none;
	background: rgb(120, 120, 120);
	padding: 3px;
}
#frameContent QComboBox:hover, #frameContent QLineEdit:hover
{
	background: rgb(170, 170, 170);
}
#frameContent QComboBox::drop-down
{
	border: none;

	subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 18px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}
#frameContent QComboBox::down-arrow
{
	image: url(:/img/1downarrow.png);
}

.ActionMenu 
{
	background-color: rgb(78, 78, 78);
	border: 1px solid black;
	min-width: 300px;
}

.ActionMenu .MenuItem
{
	min-width: 300px;
	border-bottom-width: 1px;
    border-bottom-color: rgb(72, 72, 72);
    border-bottom-style: solid;
}

.ActionMenu .MenuItem:hover
{	
	background-color: rgb(120, 120, 120);
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameHeader">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelRuleIcon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/img/icon_rule.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTitle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Condition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelRuleType">
          <property name="text">
           <string>cond_type</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/img/1rightarrow.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelRuleName">
          <property name="text">
           <string>cond_name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameContent">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelRoomIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/img/rooms/lounge_small.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRoomName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Room_name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/img/2rightarrow.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelItemName">
            <property name="text">
             <string>Item_name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLineEdit" name="editValue">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonMore">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/img/edit_add.png</normaloff>:/img/edit_add.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="class" stdset="0">
               <string>buttonStyled</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="btMore">
            <property name="text">
             <string>Dynamic value...</string>
            </property>
            <property name="class" stdset="0">
             <string>buttonStyled</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Value:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboOp"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Operator:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Use for trigger</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkTrigger">
            <property name="text">
             <string>Trigger</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
